github/repos_test.go

```diff
@@ -926,3 +926,43 @@ func TestPullRequestReviewsEnforcementRequest_MarshalJSON_nilDismissalRestirctio
 		t.Errorf("PullRequestReviewsEnforcementRequest.MarshalJSON returned %+v, want %+v", string(json), want)
 	}
 }
+
+func TestRepositoriesService_ListAllTopics(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/topics", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
```

Comment:
You could add `testHeader(t, r, "Accept", mediaTypeTopicsPreview)` here, similar to other tests.